frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -0,0 +1,177 @@
+                    endAdornment: (
+                      <>
+                        {isUsersLoading ? <Loading /> : null}
+                        {params.InputProps.endAdornment}
+                      </>
+                    ),
+                  }}
+                />
+              )}
+            />
+            <Box
+              sx={{
+                overflowY: 'auto',
+                maxHeight: '400px',
+                border: 'solid 1px',
+                padding: '20px',
+                borderColor: '#e0e0e0',
+                borderRadius: 1,
+              }}
+            >
+              <List disablePadding>
+                {accessList &&
+                  accessList.map((entity) => (
+                    <ListItem key={entity.entity}>
+                      <EntityItem entity={entity} accessList={accessList} setAccessList={setAccessList} />
+                    </ListItem>
+                  ))}
+              </List>
+            </Box>
+            <Button variant='outlined' aria-label='Save access list button' onClick={updateAccessList}>
+              Save
+            </Button>
+          </Stack>
+        </Box>
+      )}
+    </>
+  )
+}
+
+function EntityItem({
+  entity,
+  accessList,
+  setAccessList,
+}: {
+  entity: CollaboratorEntry
+  accessList: CollaboratorEntry[]
+  setAccessList: Dispatch<SetStateAction<CollaboratorEntry[]>>
+}) {
+  const roles = ['consumer', 'contributor', 'owner']
+  const fixedOptions = ['consumer']
+
+  function onRoleChange(_event: React.SyntheticEvent<Element, Event>, newValues: string[]) {
+    const updatedAccessList = _.cloneDeep(accessList)
+    const index = updatedAccessList.findIndex((access) => access.entity === entity.entity)
+    updatedAccessList[index].roles = [
+      ...fixedOptions,
+      ...newValues.filter((option) => fixedOptions.indexOf(option) === -1),
+    ]
+    setAccessList(updatedAccessList)
+  }
+
+  function removeEntity() {
+    setAccessList(accessList.filter((access) => access.entity !== entity.entity))
+  }
+
+  function rowIcon() {
+    return entity.entity.startsWith('user:') ? <PersonIcon color='primary' /> : <GroupsIcon color='secondary' />
+  }
+
+  return (
+    <Stack direction='row' justifyContent='space-between' alignItems='center' spacing={2} sx={{ width: '100%' }}>
+      <>{rowIcon()}</>
+      <Typography>{entity.entity.replace('user:', '').replace('group:', '')}</Typography>
+      <Autocomplete
+        id='role-selector'
+        sx={{ width: '100%' }}
+        size='small'
+        multiple
+        aria-label={`role selector input for entity ${entity.entity}`}
+        value={entity.roles}
+        options={roles}
+        getOptionLabel={(role) => role}
+        onChange={onRoleChange}
+        renderInput={(params) => <TextField {...params} label='Select roles' />}
+        renderTags={(value, getTagProps) =>
+          value.map((option, index) => (
+            // eslint-disable-next-line react/jsx-key
```

Comment:
We shouldn't be disabling this rule. I would take a guess that `option` would make a suitable `key`